build/known_blocks.yaml

```diff
@@ -0,0 +1,52 @@
+# This file maps epochs to block headers for calibnet and mainnet. Forest use
+# this mapping to quickly identify the origin network of a tipset.
+#
+# Block headers can be inspected on filfox:
+# - https://filfox.info/en/block/bafy2bzacebnfm6dvxo7sm5thcxnv3kttoamb53uxycnvtdxgk5mh7d73qlly2
+# - https://calibration.filfox.info/en/block/bafy2bzacedhkkz76zdekpexha55b42eop42e24ajmajm26wws4nbvtq7louvu
+#
+# This file was generated with `forest-cli archive checkpoints`
```

Comment:
new issue:

I'd like to remove the serialization for this, and turn it into rust code, probably using something like [databake](https://docs.rs/databake/latest/databake/trait.Bake.html), but that's a little heavyweight - we could then just
```rust
struct KnownBlocks<'a> {
    calibnet: &'a [(i64, Cid)],
    mainnet: &'a [(i64, Cid)],
}
const KNOWN_BLOCKS: KnownBlocks<'static> = todo!();
```

Unfortunately, multihash and cid aren't there yet:
https://github.com/multiformats/rust-cid/issues/138
https://github.com/multiformats/rust-multihash/issues/330

Reply:
#3227

Reply:
Why should it be Rust code?

Reply:
It's less work, and makes an error state impossible.
Consider this:

```rust
use std::net::Ipv4Addr;

const MEH: &str = "127.0.0.1";
const NICE: Ipv4Addr = Ipv4Addr::new(127, 0, 0, 1);

fn meh() -> Ipv4Addr {
    MEH.parse().unwrap()
}

fn nice() -> Ipv4Addr {
    NICE
}
```

Why should `meh()` need to `unwrap`, when it can be `NICE`?
`NICE` also facilitates inlining, and all that other good stuff :)